mongo/integration/mtest/mongotest.go

```diff
@@ -268,8 +268,9 @@ func (t *T) ClearMockResponses() {
 
 // GetStartedEvent returns the most recent CommandStartedEvent, or nil if one is not present.
 // This can only be called once per event.
-// TODO(GODRIVER-2075): GetStartedEvent documents that it returns the most recent event, but
-// actually returns the first event. Update either the documentation or implementation.
+//
+// TODO(GODRIVER-2075): GetStartedEvent documents that it returns the most recent event, but actually returns the first
+// TODO event. Update either the documentation or implementation.
 func (t *T) GetStartedEvent() *event.CommandStartedEvent {
```

Comment:
Same as above, move the TODO comment block to the first line of the `GetStartedEvent` function.